project/LinkerdBuild.scala

```diff
@@ -258,7 +258,12 @@ object LinkerdBuild extends Base {
        |   -Dio.netty.allocator.numDirectArenas=${FINAGLE_WORKERS:-8}    \
        |   -Dcom.twitter.finagle.netty4.numWorkers=${FINAGLE_WORKERS:-8} \
        |   ${GC_LOG_OPTION:-}                                            \
-       |   ${LOCAL_JVM_OPTIONS:-}                                        "
+       |   ${LOCAL_JVM_OPTIONS:-}"
+       |
+       |if ! [ "$LOCAL_JAVA_VERSION" -ge 9 ]; then
```

Comment:
I guess it's fine that things will break if this was ever overridden with some other than the local java version. So, should be fine.

Reply:
@adleong I'll answer your question here. I did not call this `JAVA_VERSION` because this is an environment variable that can be set in practice. I chose `LOCAL_JAVA_VERSION` because I didn't like `version`. I can change it back if we'd like!